openml/runs/functions.py

```diff
@@ -1,10 +1,10 @@
-import collections
+from collections import OrderedDict
 import io
 import json
 import os
 import sys
-import time
-from typing import List, Union, Tuple
+from typing import List, Optional, Tuple, Union  # noqa F401
+import typing
```

Comment:
Where do we make a distinction in importing the types or specifying them as `typing`? E.g. why do we use `typing.Any` but don't use `typing.List`?

Reply:
I prefer `from typing import List` etc. to have shorter type annotations. This was a mistake of mine, I'll remove the `import typing`.